lnclient/cashu/cashu.go

```diff
@@ -231,6 +220,25 @@ func (cs *CashuService) RedeemOnchainFunds(ctx context.Context, toAddress string
 }
 
 func (cs *CashuService) ResetRouter(key string) error {
+	mnemonic := cs.wallet.Mnemonic()
```

Comment:
for now we could check if `key == "ALL"` do the restore, otherwise get the mnemonic: `cashuWallet.Mnemonic()`